video_service/internal/grpcserver/grpc.go

```diff
@@ -63,7 +63,7 @@ func NewGRPCServer(bucketName string, db *sqlx.DB, port int, originFQDN string,
 	}
 
 	// TODO: context and return
-	go g.transcodeAndUploadVideos()
+	go g.transcodeAndUploadVideos(MaxDLFileSize)
```

Comment:
could also just leave this as a property of the grpcserver... but it doesn't matter.